README.md

```diff
@@ -9,6 +9,8 @@ snippets
 
 [Documentation](http://abouthome-snippets-service.readthedocs.org/)
 
+[![webqa tests](https://img.shields.io/badge/WebQA-Treeherder-lightgrey.svg)](https://treeherder.mozilla.org/#/jobs?repo=snippets-service)
```

Comment:
I would suggest treeherder:unknown as these aren't necessarily just web QA tests:

[![treeherder](https://img.shields.io/badge/treeherder-unknown-lightgrey.svg)](https://treeherder.mozilla.org/#/jobs?repo=snippets-service)

I don't know if Treeherder provides badges, but it would be a nice idea to show the number of jobs for the latest revision and how many of those are passing. Something like:

[![treeherder](https://img.shields.io/badge/treeherder-10/10-brightgreen.svg)](https://treeherder.mozilla.org/#/jobs?repo=snippets-service)

/cc @edmorley @camd